doc/License.dox

```diff
@@ -0,0 +1,32 @@
+// License Doxygen file
+
+/** \page license License
+
+<b>
+BSD 3-Clause License
+</b>
+
+<b>Copyright</b> &copy; <i>2017, 2018, ETH Zurich</i>. All rights reserved.
```

Comment:
About years for the copyright: how do we have to deal with it? I don't have any knowledge about this, but I see that here we have `2017, 2018`, while in the license header I see "2018, 2019".

How does it work? @aleksandergrm @rasolca @teonnik

Reply:
Shall we add 2018, 2019, 2020, 2021?

Reply:
I would add `2017-2021` here and in the license header.